editoast/src/views/study.rs

```diff
@@ -393,10 +427,12 @@ pub mod test {
         let response = call_service(&app, req).await;
         assert_eq!(response.status(), StatusCode::OK);
 
-        let study: Study = read_body_json(response).await;
-        assert_eq!(study.name.unwrap(), "study_test");
+        let study_response: StudyResponse = read_body_json(response).await;
+        assert_eq!(study_response.study.name.unwrap(), "study_test");
```

Comment:
I know it was already there, but this is problematic for testing isolation. If the first assert fails, the study is never deleted. Use a fixture that implements `Drop` to automatically delete the model, even when the test fails. You can then remove the explicit delete.

```suggestion
        let study_response: StudyResponse = read_body_json(response).await;
        let study = Fixture::new(study_response.study, db_pool);
        assert_eq!(study.model.name.unwrap(), "study_test");
```